src/main/java/uk/gov/hmcts/reform/cpo/service/impl/CasePaymentOrdersServiceImpl.java

```diff
@@ -64,4 +66,16 @@ private void validateCasePaymentOrderQueryFilter(final CasePaymentOrderQueryFilt
                 "case-payment-orders cannot filter case payments orders by both id and cases-id.");
         }
     }
+
+    @Override
+    public void deleteCasePaymentOrdersByIds(List<UUID> ids) throws CasePaymentIdentifierException {
+        casePaymentOrdersRepository.deleteByUuids(ids);
+        casePaymentOrdersRepository.deleteAuditEntriesByUuids(ids);
+    }
+
+    @Override
+    public void deleteCasePaymentOrdersByCaseIds(List<Long> caseIds) throws CasePaymentIdentifierException {
+        casePaymentOrdersRepository.deleteByCaseIds(caseIds);
+        casePaymentOrdersRepository.deleteAuditEntriesByCaseIds(caseIds);
```

Comment:
same as above

Reply:
as above, It is on the `CasePaymentOrdersRepository` interface